docs/topics/Gateway.md

```diff
@@ -38,26 +38,25 @@ Returns an object with the same information as [Get Gateway](#DOCS_GATEWAY/get-g
 The Discord Gateway has a versioning system which is separate from the core APIs. The following table specifies all versions of the Gateway API that have been officially supported, and whether or not they are out of service (e.g. unsupported and potentially disfunctional). The documentation herein is only for the latest version in the following table, unless otherwise specified.
 
 | Version | Out of Service |
-|------------|----------------|
+|---------|----------------|
 | 5 | no |
 | 4 | no |
 
-
 ## Gateway OP Codes/Payloads
 
 ###### Gateway Payload Structure
 
 | Field | Type | Description | Present |
-|-------|------|-------------|-------------|
+|-------|------|-------------|---------|
 | op | integer | opcode for the payload | Always |
-| d | mixed (object, integer) | event data | Always |
-| s | integer | sequence number, used for resuming sessions and heartbeats | Only for OP 0 |
-| t | string | the event name for this payload | Only for OP 0 |
+| d | ?mixed (object, integer, bool) | event data | Always |
+| s | ?integer | sequence number, used for resuming sessions and heartbeats | Always |
```

Comment:
I believe the `?` is used to show if it could be null, where the present column is used to show if the field is present in the json.